pkg/ipfailover/keepalived/generator.go

```diff
@@ -144,9 +155,10 @@ func GenerateDeploymentConfig(name string, options *ipfailover.IPFailoverConfigC
 	podTemplate := &kapi.PodTemplateSpec{
 		ObjectMeta: kapi.ObjectMeta{Labels: selector},
 		Spec: kapi.PodSpec{
-			HostNetwork: true,
-			Containers:  containers,
-			Volumes:     generateVolumeConfig(),
+			HostNetwork:  true,
+			NodeSelector: generateNodeSelector(name, selector),
```

Comment:
Not really. Maybe I wasn't clear on what the ```--label``` command was doing on the router.
But in here - for the ip failover command its solely intended for selecting the nodes where it gets placed.